tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -167,6 +169,8 @@ spec:
             | tee $(results.mustSignIndexImage.path)
         jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.publish_index_image' |tr -d "\n" \
             | tee $(results.mustPublishIndexImage.path)
+        jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.fbc_opt_in' |tr -d "\n" \
+            | tee "$(results.isFbcOptIn.path)"
```

Comment:
I think it makes sense. As you said, users have no clue why the publishing did not happen and maybe only exposing the value wouldn't help much.

I like the suggestion.

Reply:
Yes, I completely agree with the suggestion to clarify why the index image is not published for the user.

Statement from Slack thread: "When you want to have your index in production, you need the **stagedIndex: false** (or not set) and the fbcOptIn flag flipped in **pyxis**."


Do you think it looks any better? @mmalina @theflockers 
```
    if [ "$(jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.publish_index_image')" = "true" ]; then
      echo "Index image will be published."
    elif [ "$(jq -r '.genericResult | fromjson' <<< "${results}" | jq -r '.fbc_opt_in')" = "false" ]; then
      echo "Index image will not be published because FBC opt-in is set to false in Pyxis."
    elif [ "${staged_index}" = "true" ]; then
      echo "Index image will not be published because this is a staging release."
    else
      echo "Index image will not be published for an unspecified reason."
    fi
```

Reply:
I added some statements as suggested; please let me know if any corrections are needed. Thanks!

Reply:
I think the current version looks good. But I'd like @theflockers 's input as well.

Reply:
Also looks good to me.